2018 Robot/src/org/usfirst/frc/team4999/utils/MoPrefs.java

```diff
@@ -0,0 +1,64 @@
+package org.usfirst.frc.team4999.utils;
+
+import edu.wpi.first.wpilibj.Preferences;
+
+public class MoPrefs {
+	
+	private static Preferences prefs = Preferences.getInstance();
+	
+	private static final double AUTO_SPEED = 0.25; 				// Maximum speed the robot will travel at in autonomous
+	private static final double DRIVE_ENC_TICKS = 562.666378; 	// Number of encoder ticks per meter of travel
+	private static final double LIFT_ENC_TICKS = 500; 			// Number of lift encoder ticks per meter of travel
```

Comment:
Assuming this is a placeholder, or did you actually rough out some numbers based on the winch diameter?  Keep in mind that the total motion of the 1st stage will be less than 1m.

Reply:
Placeholder